sbi/src/api/attestation.rs

```diff
@@ -13,21 +14,134 @@ pub const MAX_CERT_SIZE: usize = 4096;
 /// The data blob passed to the GetEvidence call must be 64 bytes long.
 pub const REQUEST_DATA_SIZE: usize = 64;
 
-/// Attestation evidence formats.
-#[derive(Copy, Clone, Debug)]
-#[repr(u64)]
-pub enum EvidenceFormat {
-    /// Single layer DICE TCB
-    /// https://trustedcomputinggroup.org/resource/dice-attestation-architecture/
-    DiceTcbInfo = 0,
-
-    /// X.509 extension for multiple layers DICE TCB.
-    /// https://trustedcomputinggroup.org/resource/dice-attestation-architecture/
-    DiceMultiTcbInfo = 1,
-
-    /// Open DICE profile
-    /// https://pigweed.googlesource.com/open-dice/+/HEAD/docs/specification.md
-    OpenDice = 2,
+/// A list of supported hash algorithms.
+#[derive(Copy, Clone, Default, Debug)]
+#[repr(u8)]
+pub enum HashAlgorithm {
+    /// SHA-384
+    #[default]
+    Sha384 = 1,
+    /// SHA-512
+    Sha512 = 2,
+}
+
+flags! {
+    /// Attestation evidence formats.
+    #[derive(Default)]
+    #[repr(u8)]
+    pub enum EvidenceFormat: u8 {
+        /// Single layer DICE TCB
+        /// https://trustedcomputinggroup.org/resource/dice-attestation-architecture/
+        #[default]
+        DiceTcbInfo = 1,
+
```

Comment:
Nit: Redundant newline here and below.

Reply:
I personally find it more readable with those newline, but I see the rest of the code does not follow that pattern. Thanks.